scripts/postinstall.js

```diff
@@ -1,15 +1,31 @@
-const chalk = require('chalk')
+const id = (message) => message
 
-console.log('')
-console.log(chalk.greenBright.bold.underline(`Success! Netlify CLI has been installed!`))
-console.log('')
-console.log('Your device is now configured to use Netlify CLI to deploy and manage your Netlify sites.')
-console.log('')
-console.log('Next steps:')
-console.log('')
-console.log(`  ${chalk.cyanBright.bold('netlify init')}     Connect or create a Netlify site from current directory`)
-console.log(`  ${chalk.cyanBright.bold('netlify deploy')}   Deploy the latest changes to your Netlify site`)
-console.log('')
-console.log(`For more information on the CLI run ${chalk.cyanBright.bold('netlify help')}`)
-console.log(`Or visit the docs at ${chalk.cyanBright.bold('https://cli.netlify.com')}`)
-console.log('')
+const postInstall = () => {
+  let chalk
+  try {
+    // this fails sometimes on outdated npm versions
+    // eslint-disable-next-line node/global-require
+    chalk = require('chalk')
+  } catch (_) {
+    chalk = {
+      greenBright: { bold: { underline: id } },
```

Comment:
Not sure if there's a better way to do this

Reply:
If you don't want to be defining all the possible permutations of the chalk options, perhaps we could use something like:

```js
const format = (message, styles) => {
  let func = i => i
  
  try {
    func = require('chalk')
    
    styles.forEach(style => {
      func = func[style]
    })
  } catch (_) {}
  
  
  return func(message)
}
```

And then:

```js
// Instead of
chalk.greenBright.bold.underline(`Success! Netlify CLI has been installed!`)

// We'd do
format(`Success! Netlify CLI has been installed!`, ['greenBright', 'bold', 'underline'])
```